MobileCenterDistribute/MobileCenterDistribute/MSDistribute.m

```diff
@@ -539,10 +539,13 @@ - (void)showConfirmationAlert:(MSReleaseDetails *)details {
     NSString *messageFormat = details.mandatoryUpdate
                                   ? MSDistributeLocalizedString(@"MSDistributeAppUpdateAvailableMandatoryUpdateMessage")
                                   : MSDistributeLocalizedString(@"MSDistributeAppUpdateAvailableOptionalUpdateMessage");
+    NSString *appName = [MS_APP_MAIN_BUNDLE objectForInfoDictionaryKey:@"CFBundleDisplayName"];
+    if (!appName) {
+      appName = [MS_APP_MAIN_BUNDLE objectForInfoDictionaryKey:@"CFBundleName"];
+    }
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wformat-nonliteral"
```

Comment:
`NSString::stringWithFormat` recommended in Apple documentation if string resources have formatted string but `stringWithFormat` doesn't like a string variable for format string, it complains the parameter is non string literal.
If you have any recommendation, please let me know.